### PR TITLE
added theme folder and added it as a prop

### PR DESCRIPTION
**Pull request by BenBrownCreative**

### CODE REVIEW

File: client/src/components/Library/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+    return (
+        <div>
+        These are my games
+        
+        </div>
+    )
+}
+
+export default Library;
```
Comment (meyer-mcmains) [inline client/src/components/Library/index.js:15]: @BenBrownCreative something might be up with your editor settings there should always be a newline at the end of a file.

Comment (BenBrownCreative) [reply client/src/components/Library/index.js:15]: Thanks, didn't know that was a thing. I found the setting in vs code for adding a empty line to the end on save.

Comment (meyer-mcmains) [reply client/src/components/Library/index.js:15]: @BenBrownCreative Awesome, re-save these files to get the new-line and push them back up.

Comment (BenBrownCreative) [reply client/src/components/Library/index.js]: i did. what is the best way for me to get all your new changes and start working on the profile page components? do you want to merge everything you have into master now?

**Pull request merged**